components/leadership-messages/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://localhost:3000/schemas/v1.json#",
   "name": "leadership-messages",
-  "version": "2.0.8",
+  "version": "2.0.9",
```

Comment:
@broleomo @iamrentman I'm incrementing the version of Leadership Messages, but not sure if it's used in other components -  I figured you might be more familiar with this than I do. The way it's organized, I can't simply do a search for AvatarCard in the repo 🤔